types/react-datetime-picker/index.d.ts

```diff
@@ -3,8 +3,6 @@
 // Definitions by: Chris Frewin <https://github.com/princefishthrower>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import * as React from 'react';
-
```

Comment:
can you revert this part?
https://github.com/DefinitelyTyped/DefinitelyTyped#im-writing-a-definition-that-depends-on-another-definition-should-i-use-reference-types--or-an-import

It' works, but not intended behavior. I'll submit for review for CI task, it looks like a hack (because there is a test file)

Reply:
done